jwql/utils/utils.py

```diff
@@ -749,13 +749,34 @@ def read_png(filename):
 
         # Copy the RGBA image into view, flipping it so it comes right-side up
         # with a lower-left origin
-        view[:,:,:] = np.flipud(np.asarray(rgba_img))
+        view[:, :, :] = np.flipud(np.asarray(rgba_img))
     else:
         view = None
     # Return the 2D version
     return img
 
 
+def save_png(fig, filename=''):
```

Comment:
Thanks, this looks great. Is this just patching function until we can get the bokeh version updated? Or are we looking at having to keep this around long term?

Reply:
I haven't checked Bokeh > 3.0 to see, but my guess is that this will be a long term fix.